src/app/content/practiceQuestions/components/IntroScreen.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components/macro';
+import { nextQuestion } from '../actions';
+import * as pqSelectors from '../selectors';
+import PQButton from './PQButton';
+
+// tslint:disable-next-line: variable-name
+const IntroScreenWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
I think if you add `display: block` to `IntroScreenMessage `, you could get away with just putting `width: 100%; and text-align: center;` to make it simpler